### PR TITLE
Add environment variables to project

### LINKED ISSUE #937: Set up environment variables
Adding environment variables will allow us to differentiate between local development and the live, published site. This is great for things like ensuring our analytics are accurate, as well as better CI/CD integration (not to mention clever things like creating favicons to indicate if the site is live or not when working on it).

Here's a good starting point on setting this up in Eleventy: https://weblog.herok.pl/posts/eleventy-tip-use-environment-variables/

### PR DESCRIPTION
**Pull request by danielmatthew**
A first pass. In #937 I'd seen reference to #966, so in development mode we don't pull in the service worker partial, nor the analytics. 

Before:
```html
<head>
  <script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script>
  <script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-2419836-9"></script>
  <script>…</script>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <meta name="viewport" content="width=device-width,initial-scale=1">
  [ Truncated… ]
  <link rel="sitemap" type="application/xml" title="The A11Y Project" href="/sitemap.xml"><link rel="author" href="humans.txt">
  <script async="">if ('serviceWorker' in navigator) {
		window.addEventListener('load', () => {
			navigator.serviceWorker.register("/sw.js");
		});
	}
  </script>
</head>
```

After:

```html
<head>
  <meta charset="utf-8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <meta name="viewport" content="width=device-width,initial-scale=1">
   [ Truncated… ]
  <link rel="sitemap" type="application/xml" title="The A11Y Project" href="/sitemap.xml"><link rel="author" href="humans.txt">
</head>
```

Closes #937

### CODE REVIEW

**Comment by danielmatthew** [comment]
Ugh… that merge conflict :/

File: src/_includes/layouts/base.njk [hunk 1]
```diff
@@ -9,8 +12,10 @@
 		<link rel="stylesheet" as="style" href="{{ '/css/screen.min.css?v=' + getCurrentVersion.version  | url }}" />
 		<link rel="alternate" href="{{ metadata.feed.path | url }}" type="application/atom+xml" title="{{ metadata.title }}" />
 		<link rel="sitemap" type="application/xml" title="{{ metadata.title }}" href="{{ '/sitemap.xml' | url }}" />
-		<link rel="author" href="/humans.txt" />
-		{% include "meta/service-worker.njk" %}
+		<link rel="author" href="humans.txt" />
```
Comment (danielmatthew) [inline src/_includes/layouts/base.njk:15]: Ugh, sorry, I hadn't spied the forward slash when I fixed a conflict.

File: src/_data/env.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require('dotenv').config();
+
+module.exports = {
+  environment: process.env.ELEVENTY_ENV || 'development'
```
Comment (danielmatthew) [inline src/_data/env.js:4]: This would need to be set to something other than 'development' over on Netlify…

**Comment by SaptakS** [comment]
@danielmatthew Thanks for the work! I reviewed the code and it works perfectly to include the service worker and the analytics only if the environment variable is not labelled development. I think we can add a bit more things to the envionment variables. For example, a lot of the information in `src/_data/metadata.json` I feel should be configurable by environment variables, like the url, logo, title, etc. Other than that, looks good to me.

Also, you will need to resolve the merge conflict again.

**Pull request closed without merging**